apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/Form/Rates/DecentralisedRate.jsx

```diff
@@ -22,13 +28,9 @@ function DecentralisedRate({ value = {}, selected, onSelect, icon }: Props) {
       title={value.name}
       subtitle={<Trans i18nKey={"swap2.form.rates.noRegistration"} />}
       rightContainer={
-        <Text
-          fontSize={3}
-          color="palette.text.shade40"
-          style={{ width: "110px", textAlign: "right" }}
-        >
+        <SecondaryText fontSize={3} style={{ width: "110px", textAlign: "right" }}>
```

Comment:
It's better to use the design system font variant :
variant="body" (or any corresponding variant) instead of fontSize={3}
https://react-ui-storybook.vercel.app/?path=/story/asorted-typography-text--overview&args=variant:h5

Reply:
(But you need to import Text for react-ui)

Reply:
I have move the import origin to core, although the size is not the correct.
I have try with "small" variant, but always have 13px, not 12px. (body have 13 px too)